### ISSUE TITLE
Going Underground

### ISSUE THREAD

**Issue by rdixon22** [decentraland/proposals#74]
Underground worlds are popular settings for many video games, books and films. Many land owners would want to create underground environments if they could. Examples include: dungeons, mining tunnels, secret passages, and so on.

In the current Decentraland prototype everything is above ground. It would be great to allow underground areas as well, so people could build above and below the ground line.

The underground area would not have to extend as far as the above ground area does (are there even any limits to the vertical dimeension yet?). Allowing an extre 20m or 30m depth might be enough. This could be empty space with a second floor at the bottom -- it would not have to be filled with dirt that you must extract, as in Minecraft.

Of course, there would have to be a way to cut an opening in the main surface to allow people to get to the underground space.

Without an official underground space, people could still simulate underground environments by creating sloped terrain on their land, to let players climb up, and then building "underground" spaces beneath that terrain. But it would require  a lot of contiguous terrain to do this well. 

Given a maximum 45 degree slope, you would about 5m width to get up to 5m height (about the minumum you'd need to have enough headspace underground). So you would need at least 4 plots of land (2 x 2) to create even a small hill with an underground area. If everyone could build 20m or 30m below ground by default, this would be much easier.

**Comment by rdixon22** [comment]
It did not occur to me until after I posted this, but there is another great use for an underground space: a subway system.  Plots of land could be reserved for Stations, and then a subway system could be built in the underground area, connecting the stations. This would greatly reduce the amount of above-ground land that needs to be dedicated to transportation systems.

**Comment by Dr-Doer** [comment]
I agree. To not include a decent amount of underground space takes away a lot of use cases, profit, and freedom for everyone. There are multiple uses besides just transportation or those type uses but also being able to build buildings underground and without the issues you might run into by users trying to build dirt above ground to bury things under and making a weird landscape and limited possibilities.
